slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -226,7 +226,10 @@ async def receive_messages(self) -> None:
                             if message.data is not None:
                                 str_message_data = message.data.decode("utf-8")
                                 elements = str_message_data.split(":")
-                                if len(elements) == 2:
+                                if (
+                                    len(elements) == 2
+                                    and elements[0] == "sdk-ping-pong"
```

Comment:
This is a minor improvement for https://github.com/slackapi/python-slack-sdk/pull/1112